src/client/datascience/kernel-launcher/localKernelFinder.ts

```diff
@@ -122,17 +159,71 @@ export class LocalKernelFinder implements ILocalKernelFinder {
         ]);
 
         const kernels = this.filterKernels(nonPythonKernelSpecs.concat(pythonRelatedKernelSpecs));
-        sendKernelListTelemetry(resource, kernels);
+        this.lastFetchedKernelsWithoutCache = kernels;
+        this.globalState.update('JUPYTER_GLOBAL_KERNELSPECS', kernels).then(noop, (ex) => {
```

Comment:
Probably makes sense to turn this string into a constant so we don't have any typos in other uses.

Reply:
oops